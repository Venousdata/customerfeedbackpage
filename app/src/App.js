import { BrowserRouter, Switch, Route } from "react-router-dom";
import { makeStyles, Paper } from "@material-ui/core";
import Home from "./components/Home";
import Product from "./components/Product";

const useStyles = makeStyles((theme) => ({
  paper: {
    padding: "2em",
  },
  pagecontainer: {
    position: "relative",
    minHeight: "100vh",
    textAlign: "center",
  },
  contentwrap: {
    padding: "1rem",
  },
}));

const App = () => {
  const classes = useStyles();

  return (
    <div className={classes.pagecontainer}>
      <BrowserRouter>
        <div className={classes.contentwrap}>
          <Paper elevation={2} className={classes.paper}>
            <h1>Customer Feedback Page</h1>
            <Switch>
              <main>
                <Route path="/" exact component={Home} />
                <Route path="/product/:id" exact component={Product} />
              </main>
            </Switch>
          </Paper>
        </div>
      </BrowserRouter>
    </div>
  );
};

export default App;
