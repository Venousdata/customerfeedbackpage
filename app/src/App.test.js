import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App tests", () => {
  it("App should render and match snapshot", (done) => {
    const container = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
    done();
  });
});
