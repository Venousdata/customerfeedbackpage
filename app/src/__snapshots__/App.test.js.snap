// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests App should render and match snapshot 1`] = `undefined`;
